bubble_sort/Python3/mergeSort.py

```diff
@@ -0,0 +1,16 @@
+def bubbleSort(arr):
+    n = len(arr)
+ 
+    # Traverse through all array elements
+    for i in range(n):
+ 
+        # Last i elements are already in place
+        for j in range(0, n-i-1):
+ 
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpyaw51koo/bubble_sort/Python3/mergeSort.py
+++ b/tmp/tmpyaw51koo/bubble_sort/Python3/mergeSort.py
@@ -6,7 +6,7 @@
  
         # Last i elements are already in place
         for j in range(0, n-i-1):
- 
+
             # traverse the array from 0 to n-i-1
             # Swap if the element found is greater
             # than the next element
```